cmds/webboot/webboot.go

```diff
@@ -16,10 +20,12 @@ var (
 	v       = flag.Bool("verbose", false, "Verbose output")
 	verbose = func(string, ...interface{}) {}
 	dir     = flag.String("dir", "", "Path of cached directory")
-	network = flag.Bool("network", true, "Should or not set up network")
-	boot    = flag.Bool("boot", true, "Should or not boot the iso. May trun it off for test.")
+	network = flag.Bool("network", true, "If network is false we will not set up network")
+	dryRun  = flag.Bool("dry_run", true, "If dry_run is true we won't boot the iso.")
 )
 
+var cacheDir = ""
```

Comment:
Is this necessary anymore? Seems like you pass a parameter cacheDir anyway

Reply:
hmm I need this at the line 225 for check if the current dit of DirOption is the cache dir itself. if it is I should back to the main menu instead of go to it's father folder

Reply:
nevertheless the zero value of a string is ""
var cacheDir
is the same as 
var cacheDir=""

Reply:
get it! thanks

Reply:
Actually I'm not sure that this variable is being used anymore, there's also a cacheDir in the main function which seems to be what you're using, right?

Reply:
yeah I did not need it anymore... thank you Prachi!